src/EntityGetter/EntityGetter.js

```diff
@@ -1,40 +1,20 @@
-import { clone, each, filter, first, get as _get, values } from 'lodash';
+import { get as _get, values } from 'lodash';
```

Comment:
this can't be imported as `get` because there's already a function called `get` on the `EntityGetter` class.